recipes/xsel/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "xsel" %}
+{% set version = "1.2.1" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kfish/xsel/archive/{{ version }}.tar.gz
+  sha256: 18487761f5ca626a036d65ef2db8ad9923bf61685e06e7533676c56d7d60eb14
+
+build:
+  number: 0
+  skip: true  # [not unix]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - make
+    - pkg-config  # [unix]
+    - {{ stdlib('c') }}             # [unix]
+    - {{ compiler('c') }}           # [unix]
+    - make                          # [unix]
+    - autoconf                      # [unix]
+    - automake                      # [unix]
+    - gettext                       # [unix]
+    - libtool                       # [unix]
+    - xorg-libxt

```

Comment:
shouldn't this be in the host section?

Reply:
They're build dependencies as per Homebrew and Arch.

Reply:
https://formulae.brew.sh/formula/xsel

https://archlinux.org/packages/extra/x86_64/xsel/

Reply:
Even if it just needs the headers, it should go into host. Different tools/ecosystems don't necessarily agree on all details of naming/semantics, but in conda-forge, the build environment is restricted purely to things that need to be executed during the build (like a compiler); everything you want to compile _against_, whether shared, static or header-only, needs to go into host.

Reply:
Moved to headers and instead put in ignore_run_export then.

Reply:
@h-vetinari